resource_sample_py/rockcraft.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: temporal-worker
+summary: Temporal worker app
+description: OCI image for Temporal worker app
+version: "1.0"
+base: ubuntu@22.04
+build-base: ubuntu@22.04
+license: Apache-2.0
+
+services:
+  temporal-worker:
```

Comment:
Will come back to review this properly but I had a fix an issue caused by JIMM's rock which might help save you some time, I recommend giving this PR a read https://github.com/canonical/jimm/pull/1247

Reply:
There's a recommendation from the ROCKs team to include this so that the ROCK can be deployed without the charm. It should be fine here because the charm overrides this, but it's useful to have so users can verify the ROCK has been built correctly and functions as intended. (cc @AmberCharitos )